upcgen.sh

```diff
@@ -24,6 +24,7 @@ mkdir -p $INSTALLROOT/bin
 cp ./upcgen $INSTALLROOT/bin/
 mkdir -p $INSTALLROOT/lib
 cp ./libUpcgenlib.a $INSTALLROOT/lib/.
+cp ./libUpcgenlib.so $INSTALLROOT/lib/.
 cp -r $SOURCEDIR/include $INSTALLROOT/.
 
 #ModuleFile
```

Comment:
While you are at it, please use `alibuild-generate-module --lib --bin` as done elsewhere.

Reply:
No need for the custom PATH / LD_LIBRARY_PATH. I removed it.